.github/workflows/linux_builds.yml

```diff
@@ -117,10 +117,10 @@ jobs:
         uses: ./.github/actions/godot-deps
 
       - name: Set up .NET Sdk
-        uses: actions/setup-dotnet@v3
+        uses: actions/setup-dotnet@v4
         if: ${{ matrix.build-mono }}
         with:
-          dotnet-version: '6.0.x'
+          dotnet-version: '8.0.x'
```

Comment:
I wonder if we should just remove this step since the image we use (`ubuntu-20.04`) already includes the .NET SDK[^1] so this step is just ignored, and the version that comes with the image is used instead (which was already 8.0).

See https://github.com/godotengine/godot/issues/86374.

[^1]: https://github.com/actions/runner-images/blob/main/images/ubuntu/Ubuntu2004-Readme.md#net-tools

Reply:
Ah, I was wondering where the runtime 6 was coming from. Your link explains it.